src/lib/components/drip-list-card/drip-list-card-thumblink.svelte

```diff
@@ -38,38 +38,59 @@
     SPLITS_COMPONENT_ADDRESS_RECEIVER_FRAGMENT,
     SPLITS_COMPONENT_DRIP_LIST_RECEIVER_FRAGMENT,
     SPLITS_COMPONENT_PROJECT_RECEIVER_FRAGMENT,
+    type SplitsComponentSplitsReceiver,
   } from '../splits/splits.svelte';
   import balancesStore from '$lib/stores/balances/balances.store';
   import AggregateFiatEstimate from '../aggregate-fiat-estimate/aggregate-fiat-estimate.svelte';
   import { constants } from 'radicle-drips';
   import { PROJECT_AVATAR_FRAGMENT } from '../project-avatar/project-avatar.svelte';
   import Pile from '../pile/pile.svelte';
-  import { fade } from 'svelte/transition';
   import { browser } from '$app/environment';
   import TextExpandable from '../text-expandable.svelte/text-expandable.svelte';
   import mergeAmounts from '$lib/utils/amounts/merge-amounts';
   import accountFetchStatusses from '$lib/stores/account-fetch-statusses/account-fetch-statusses.store';
   import getIncomingSplitTotal from '$lib/utils/splits/get-incoming-split-total';
-  import ChevronRight from '$lib/components/icons/ChevronRight.svelte';
   import type { DripListCardThumblinkFragment } from './__generated__/gql.generated';
   import getIncomingGivesTotal from '$lib/utils/gives/get-incoming-gives-total';
-  import { onMount } from 'svelte';
+  import { onDestroy, onMount } from 'svelte';
   import streamsStore from '$lib/stores/streams/streams.store';
   import getSupportersPile, {
     DRIP_LIST_CARD_SUPPORTER_PILE_FRAGMENT,
   } from './methods/get-supporters-pile';
+  import type { VotingRound } from '$lib/utils/multiplayer/schemas';
+  import VotingRoundSplits from './components/voting-round-splits.svelte';
+  import assert from '$lib/utils/assert';
+  import StatusBadge from '../status-badge/status-badge.svelte';
+  import Proposals from '../icons/Proposals.svelte';
 
-  export let dripList: DripListCardThumblinkFragment;
+  export let dripList: DripListCardThumblinkFragment | null = null;
+  export let votingRound: (VotingRound & { splits?: SplitsComponentSplitsReceiver[] }) | null =
+    null;
 
-  $: listOwner = dripList.owner;
-  $: dripListUrl = `/app/drip-lists/${dripList.account.accountId}`;
+  onMount(() => {
```

Comment:
pretty sure this doesn't need `onMount`?

The assert should be either just in top level, or even better in a reactive block so that it also throws if the props are later changed to an invalid state.